docker.nix

```diff
@@ -0,0 +1,99 @@
+{ dockerTools, backend, frontend, runCommand, writeText, writeScriptBin, bash, cacert, gettext, iana_etc, caddy }:
+let
+  inherit (dockerTools)
+    buildImage
+    buildLayeredImage
+    pullImage
+    shadowSetup;
+
+  inherit (builtins) toJSON;
+
+  httpPort = 80;
+  httpsPort = 443;
+
+  # Some system files necessary for nginx to function normally
+  shadow-files = runCommand "shadow-files" {} ''
+    mkdir -p $out/etc
+    echo "nobody:x:65534:65534:nobody:/:/sbin/nologin" > $out/etc/passwd
+    echo "nobody:!x:::::::" > $out/etc/shadow
+    echo "nobody:x:65534:" > $out/etc/group
+    echo "nobody:x::" > $out/etc/gshadow
+  '';
+
+  backend-config-template = writeText "backend-config-env.yaml.template" (toJSON {
```

Comment:
BTW you can import the `config.yaml` that's in the repo here and merge it with those overrides. See https://gitlab.com/tezosagora/agora/-/blob/master/nix/modules/services/backend.nix#L61 for example

Reply:
Oh, nice!